src/styles.css

```diff
@@ -24,6 +20,20 @@ button {
   margin: 8px; 
 }
 
+button.medium {
+  font-size: 18px;
+  font-weight: 500;
+  height: 48px;
+  padding: 10px 25px;
+}
+
+button.large {
+  font-size: 22px;
+  font-weight: 800;
+  height: 54px;
+  padding: 12px 27px;
+}
+
```

Comment:
I know that I originally added the styles to all generic button tags, but it is actually very very bad practice to add global styles to elements 🙀 (shame on u lynn). Also, these classnames are too generic. It's best practice to be verbose with classnames in global css files to avoid accidentally overriding classes!

Ideally, you want to minimize the css that goes into this file, as css files tend to grow fast and become impossible to maintain. Since the Button component is styled dynamically, it would be a good idea to move this into the same or adjacent file as Button. You can pass styles to components as an object with the [style attribute](https://reactjs.org/docs/dom-elements.html#style), although they make it clear that this method is not recommended.

A more elegant solution would be using the CSS-in-JS pattern. Two most popular CSS-in-JS libraries are [styled-components](https://styled-components.com/) and [emotion](https://emotion.sh/docs/introduction). I'm pretty agnostic to either. Here's an interesting [comparison between the two](https://github.com/jsjoeio/styled-components-vs-emotion).

Reply:
Thanks for the detailed explanation! I read the comparison you linked to and the differences sound pretty minimal. `styled-components` has a larger community, whereas `emotion` is lighter weight and more performant. One additional factor is that Apartment List uses `emotion`, so I will work on using that library in this PR.

Reply:
FYI - I had to set the jsx pragma [according to the documentation](https://emotion.sh/docs/css-prop#jsx-pragma) because `create-react-app` does not allow us to configure babel.